QuickBytes/src/main/java/com/quickbytes/repository/RequestRepository.java

```diff
@@ -0,0 +1,27 @@
+package com.quickbytes.repository;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+
```

Comment:
If having a consistent path variable matters, review this.

Reply:
Above